test/controllers/graphql_controller_12_test.rb

```diff
@@ -731,4 +731,119 @@ def teardown
     assert_equal 1, pm_tags.count
     assert_equal 1, fc_tags.count
   end
+
+  test "should append FactCheck to ProjectMedia, if ProjectMedia already exists and does not have a FactCheck" do
+    Sidekiq::Testing.inline!
+    url = 'http://example.com'
+    t = create_team
+    p = create_project team: t
+    pm = create_project_media team: t, url: url
+
+    assert_not_nil pm
+
+    a = ApiKey.create!
+    b = create_bot_user api_key_id: a.id
+    create_team_user team: t, user: b
+    authenticate_with_token(a)
+
+    pender_url = CheckConfig.get('pender_url_private') + '/api/medias'
+    response_body = '{"type":"media","data":{"url":"' + url + '","type":"item"}}'
+    WebMock.stub_request(:get, pender_url).with({ query: { url: url } }).to_return(body: response_body)
+
+    query = <<~GRAPHQL
+      mutation {
+        createProjectMedia(input: {
+          project_id: #{p.id},
+          media_type: "Blank",
+          channel: { main: 1 },
+          set_tags: ["tag"],
+          set_status: "verified",
+          set_claim_description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+          set_original_claim: "#{url}",
+          set_fact_check: {
+            title: "Title #1",
+            language: "en",
+          }
+        }) {
+          project_media {
+          dbid
+          full_url
+            claim_description {
+              fact_check {
+                dbid
+              }
+            }
+          }
+        }
+      }
+    GRAPHQL
+
+    post :create, params: { query: query, team: t.slug }
```

Comment:
I suggest we put a `assert_no_difference 'ProjectMedia.count'` and a `assert_difference 'FactCheck.count'` around this line.